crate_universe/defs.bzl

```diff
@@ -6,9 +6,9 @@ load("//rust:defs.bzl", "rust_common")
 load("//rust:repositories.bzl", "DEFAULT_TOOLCHAIN_TRIPLES")
 load("//rust/platform:triple_mappings.bzl", "system_to_binary_ext", "triple_to_system")
 
-DEFAULT_CRATE_REGISTRY_TEMPLATE = "https://crates.io/api/v1/crates/{crate}/{version}/download"
+DEFAULT_DEFAULT_CRATE_DOWNLOAD_URL_TEMPLATE = "https://crates.io/api/v1/crates/{crate}/{version}/download"
```

Comment:
`DEFAULT_DEFAULT`?

Reply:
Intentional (but definitely looks weird!)

`DEFAULT_CRATE_DOWNLOAD_URL_TEMPLATE` is the crate download URL template for crates from the default registry (i.e. crates.io)

This is the default value for that

Reply:
Maybe add a comment above this then?

Reply:
Fair - added comment :)